specs/electra/validator.md

```diff
@@ -220,4 +221,30 @@ with updated field assignments:
 
 - Set `attestation_data.index = 0`.
 - Let `aggregation_bits` be a `Bitlist[MAX_VALIDATORS_PER_COMMITTEE * MAX_COMMITTEES_PER_SLOT]` of length `len(committee)`, where each bit set from each individual attestation is set to `0b1`.
-- Set `attestation.committee_bits = committee_bits`, where `committee_bits` has the bit set corresponding to `committee_index` in each individual attestation.
+- Set `attestation.committee_index = committee_index`, where `committee_index` is the `committee_index` in each individual attestation.
+
+#### Aggregate signature
+
+Set `aggregate_attestation.signature = aggregate_signature` where `aggregate_signature` is obtained from:
+
+```python
+def get_aggregate_signature(attestations: Sequence[CommitteeAttestation]) -> BLSSignature:
+    signatures = [attestation.signature for attestation in attestations]
+    return bls.Aggregate(signatures)
+```
+
+### Broadcast aggregate
+
+`get_aggregate_and_proof` is modified to accept `CommitteeAttestation` for `aggregate`.
+
+```python
+def get_aggregate_and_proof(state: BeaconState,
+                            aggregator_index: ValidatorIndex,
+                            committee_attestation: CommitteeAttestation,
+                            privkey: int) -> AggregateAndProof:
+    return AggregateAndProof(
+        aggregator_index=aggregator_index,
+        aggregate=committee_attestation,
+        selection_proof=get_slot_signature(state, aggregate.data.slot, privkey),
+    )
+```
```

Comment:
I think these two sections are trivial and wordy and unrelated to the change, right? I would say they are not needed.
```suggestion
```